src/js/common/schemaform/FormStartControls.jsx

```diff
@@ -55,11 +55,11 @@ class FormStartControls extends React.Component {
               onButtonClick={this.handleLoadForm}
               buttonText="Resume previous application"
               buttonClass="usa-button-primary"/>
-          <ProgressButton
+          {!this.props.resumeOnly && <ProgressButton
               onButtonClick={this.toggleModal}
               buttonText="Start over"
```

Comment:
I wonder if the start over button should be on the left and the resume previous application button should be on the right with the chevrons?